web/assets/dropdown-menu.js

```diff
@@ -1,4 +1,22 @@
+import MOJFrontend from "@ministryofjustice/frontend/moj/all.js";
+
 export default function dropdownMenu() {
+  new MOJFrontend.ButtonMenu({
+    container: $('[data-module="app-button-menu"]'),
+    mq: "(min-width: 200em)",
+    buttonText: "Menu",
+    buttonClasses:
+      "govuk-button--secondary moj-button-menu__toggle-button--secondary",
+  });
+
+  new MOJFrontend.ButtonMenu({
+    container: $('[data-module="button-menu"]'),
+    mq: "(min-width: 200em)",
+    buttonText: "Menu",
+    buttonClasses:
+      "govuk-button--secondary moj-button-menu__toggle-button--secondary",
+  });
```

Comment:
This isn't needed if you remove the attributes from the documents/payments files